src/gc/unix/cgroup.h

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*++
+
+Module Name:
+
+    cgroup.h
+
+Abstract:
+    Read memory limits for the current process
+--*/
+
+#define MAX_SIZE_T (~(SIZE_T)0)
```

Comment:
It seems that these defines can be in the cgroup.cpp since they are not part of the public interface of the CGroup.